codewof/templates/users/activity_graph.html

```diff
@@ -4,17 +4,17 @@
   <style>
   .grid-container {
     display: grid;
-    grid-gap: 20px;
+    grid-gap: 20rem;
     grid-template-columns: auto auto auto auto auto auto auto;
     background-color: #FFFFFF;
-    padding: 10px;
+    padding: 10rem;
   }
 
   .grid-item {
     background-color: rgba(0, 0, 255, 0.8);
     border: 1px solid rgba(0, 0, 0, 0.8);
-    padding: 20px;
-    font-size: 30px;
+    padding: 20rem;
+    font-size: 30rem;
```

Comment:
Note that 1rem is ~16px, rem is suggested because it is variable, rather than absolute

https://www.sitepoint.com/understanding-and-using-rem-units-in-css/